content/en/docs/installing-on-kubernetes/hosted-kubernetes/install-ks-on-huawei-cce.md

```diff
@@ -29,16 +31,19 @@ After you get the configuration file for kubectl, use kubectl command lines to v
 $ kubectl version
 Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.8", GitCommit:"9f2892aab98fe339f3bd70e3c470144299398ace", GitTreeState:"clean", BuildDate:"2020-08-15T10:08:56Z", GoVersion:"go1.14.7", Compiler:"gc", Platform:"darwin/amd64"}
 Server Version: version.Info{Major:"1", Minor:"17+", GitVersion:"v1.17.9-r0-CCE20.7.1.B003-17.36.3", GitCommit:"136c81cf3bd314fcbc5154e07cbeece860777e93", GitTreeState:"clean", BuildDate:"2020-08-08T06:01:28Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
-
 ```
 
 ## KubeSphere Deployment
 
 ### Create a custom StorageClass
 
-> Huawei CCE built-in Everest CSI provides StorageClass `csi-disk` which uses SATA (normal I/O) by default, but the actual disk that is for Kubernetes clusters is either SAS (high I/O) or SSD (extremely high I/O). So it is suggested that create an extra StorageClass and set it as default for later. Refer to the official document - [Use kubectl to create a cloud storage](https://support.huaweicloud.com/en-us/usermanual-cce/cce_01_0044.html).
+{{< notice note >}}
```

Comment:
Is {{< notice note >}} working on ks web page? It shows as it is in GitHub page

Reply:
This format is working on KubeSphere website while not applicable in GitHub.